predictsignauxfaibles/utils.py

```diff
@@ -1,6 +1,9 @@
 import logging
 from typing import NamedTuple, List
 
+import importlib.util
+from pathlib import Path
```

Comment:
Minor detail but please look at the [PEP8 rule for import order](https://www.python.org/dev/peps/pep-0008/#:~:text=Imports%20are%20always%20put%20at,Standard%20library%20imports.) :

> Imports should be grouped in the following order:
> - Standard library imports.
> - Related third party imports.
> - Local application/library specific imports.
>
> You should put a blank line between each group of imports.